test-data/run.test

```diff
@@ -158,3 +158,23 @@ from native import f
 print(f((1,2)))
 [out]
 (1, 2)
+
+[case testTupleGet]
+from typing import Tuple
+def f(x: Tuple[Tuple[int, bool], int]) -> int:
+    return x[0][0]
+[file driver.py]
+from native import f
+print(f(((1,True),2)))
+[out]
+1
+
+[case testNewTuple]
+def f() -> int:
+    x = (False, 1)
```

Comment:
Test case where the second item is a boxed tagged integer (can't use a literal for that yet -- pass it as an argument perhaps).